modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -93,6 +93,34 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
 # ...
 ----
 
+[id="virt-example-nmstate-IP-management-mac_{context}"]
+== Media Access Control (MAC) address
+
+You can use a MAC address to identify a network interface instead of using the name of the network interface. A network interface name can change for various reasons, such as an operating system configuration change. However, every network interface has a unique MAC address that does not change. This means that using a MAC address is a more permanent way to identify a specific network interface.
+
+Supported values for the `identifier` parameter include the default `name` value and the value `mac-address`. The `name` value applies a configuration to an interface that holds a specified interface name. 
+
+Using a `mac-address` value for the `identifier` parameter indicates that a MAC address is the identifier for the network interface. If you set the `identifier` value to `mac-address`, you must enter a specific MAC address in the following `mac-address` parameter field. 
+
+[NOTE]
+====
+You can still specify a value for the `name` parameter, but setting the `identifier: mac-address` value means that a MAC address is used as the primary identifier for a network interface. If you specify an incorrect MAC address, `nmstate` reports an invalid argument error. 
+====
+
+The following snippet specifies a MAC address as the primary identifier for an Ethernet device, named `eth1`, with a MAC address of `8A:8C:92:1A:F6:98`:
+
+[source,yaml]
+----
+# ...
+interfaces:
+- name: eth1
+  profile-name: wan0
+  type: veth
```

Comment:
Sorry I didn't pick up on this earlier, but why is this a veth? I think type: ethernet would be more common for this type of configuration. I'm not actually sure this makes sense in the context of a veth since that would be a virtual interface not necessarily associated with a physical MAC address.

Reply:
My bad, I copied this configuration from another file. I updated the configuration to now state `type: ethernet`